Gruntfile.js

```diff
@@ -41,20 +41,19 @@ module.exports = function(grunt) {
         target: ['src/*.js']
     },
     run: {
-      npm_test_jest: {
+      npm_jest: {
         cmd: 'npm',
         args: [
           'run',
           'test',
           '--silent'
         ]
       },
-      npm_test_jest_cov: {
+      npm_jest_cov: {
```

Comment:
So discovered that jest has [istanbul](https://istanbul.js.org/docs/tutorials/jest/) built in.